pom.xml

```diff
@@ -59,6 +59,20 @@
       <artifactId>json</artifactId>
       <version>20200518</version>
     </dependency>
+
+    <dependency>
```

Comment:
Are we using slf4j for logging?

Reply:
Looks like we use slf4j in the AwsDepositor file, must have been left out somehow...strange.